### PR TITLE
don't wrap and re-throw, pollutes logs and causing exception is handl…

### PR DESCRIPTION
**Pull request by jflo**
…ed correctly

Signed-off-by: Justin Florentine <justin+github@florentine.us>

### CODE REVIEW

File: ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/AbstractBlockCreator.java [hunk 1]
```diff
@@ -208,8 +208,6 @@ protected Block createBlock(
       return new Block(blockHeader, new BlockBody(transactionResults.getTransactions(), ommers));
     } catch (final SecurityModuleException ex) {
       throw new IllegalStateException("Failed to create block signature", ex);
-    } catch (final CancellationException ex) {
-      throw new IllegalStateException("Attempt to create block was interrupted", ex);
```
Comment (ajsutton) [inline ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/AbstractBlockCreator.java:212]: Won't this now be caught by the next `catch (Exception` block and wrapped in `IllegalStateException` anyway?

**Review by ajsutton — APPROVED** [review]
LGTM.

**Comment by ajsutton** [comment]
Pending unit tests passing obviously :)

**Comment by shemnon** [comment]
qostimer has been flakey for me.  Re-runnung.

**Pull request merged**